packages/shared/src/graphql/notifications.ts

```diff
@@ -83,3 +83,11 @@ export const READ_NOTIFICATIONS_MUTATION = gql`
     }
   }
 `;
+
+export const NEW_NOTIFICATIONS_SUBSCRIPTION = gql`
```

Comment:
@idoshamun Can you check if this is correct? I got this from here:
https://github.com/dailydotdev/daily-api/blob/2bc1d3602f264f544f5b51d2a2aa31309bae5b21/src/schema/notifications.ts#L266